### ISSUE TITLE
Issues with PHP 7.1 RC3

### ISSUE THREAD

**Issue by prolic** [pdezwart/php-amqp#253]
Running the test-suite of https://github.com/prolic/HumusAmqp with PHP 7.1 RC3 gives test-failures that were not present with 7.0.8.

I need to make some investigations into this, but maybe you can verify this by adding 7.1 to the travis test suite and find some stuff yourself.

**Comment by prolic** [comment]
I am about to compile PHP 7.1 RC5 and will test with this. Will give you updates as soon as I can.

**Comment by prolic** [comment]
Your test suite fails on my machine with PHP 7.1 RC5:

```
FAILED TEST SUMMARY
---------------------------------------------------------------------
Bug 19840: Connection Exception [tests/bug_19840.phpt]
#147 Segfault when catchable fatal error happens in consumer [tests/bug_gh147.phpt]
Compare version in package.xml and module [tests/package-version.phpt]
```

Problems with https://github.com/prolic/HumusAmqp also still persists (two failing test-cases) - will investigate into those asap.

**Comment by pinepain** [comment]
Confirmed, build fails (https://travis-ci.org/pinepain/php-amqp/jobs/170544567), will fix it now.

**Comment by pinepain** [comment]
ptal https://github.com/pdezwart/php-amqp/pull/254

**Comment by pinepain** [comment]
@prolic can you please try current master and see whether the issue resolved by #254 and whether your failed tests on 7.1 relevant to php-amqp?

**Comment by prolic** [comment]
confirmed, all works now
